DebugLog/Manager.h

```diff
@@ -401,27 +401,6 @@ namespace debug {
                 case LogLevel::LVL_TRACE: header = "[TRACE] "; break;
                 default: header = ""; break;
             }
-#ifdef ARDUINO
-            if (b_file) header += file + string_t(" ");
-            if (b_line) header += string_t("L.") + line + string_t(" ");
-            if (b_func) header += func + string_t(" ");
```

Comment:
`b_file`, `b_line`, `b_func` are not used now? If so, it is good to remove them.

Reply:
I agree, should be good to remove

Reply:
Could you remove `b_file`, `b_line`, `b_func` and `option()` as well?

Reply:
Made changes and tested on my device. Works on my end.